### DISCUSSION TITLE
Pair programming

### DISCUSSION THREAD

**Discussion by ar-g** [MobileNativeFoundation/discussions#94] [General]
Does anyone practice pair programming at their mobile teams? If so how do you pair within a team spread across different time zone?

**Comment by keith** [comment]
There's a bit about this here as well https://github.com/MobileNativeFoundation/discussions/discussions/9#discussioncomment-423336

**Comment by maciejpiotrowski89** [comment]
I worked for 1.5 year in a company that had extreme programming [XP] practices deeply integrated in its working culture (including TDD and pair programming). I recorded a podcast episode where I talk with my former boss about pair-programming experience. I hope it will give you some insights on the topic.

[Podcast Episode about Pair Programming - "Sharing the experience"](https://diversityin.tech/2020/09/11/Pair-Programming-Mike-Mazur.html)
